roles/confluent.connect-distributed/defaults/main.yml

```diff
@@ -6,6 +6,8 @@ kafka:
       systemd_override: /etc/systemd/system/confluent-kafka-connect.service.d
       user: cp-kafka-connect
       group: confluent
+      keytab: /etc/security/keytabs
```

Comment:
keytab dir should be a variable because confluent.kerberos _luckily_ also synchronizes to /etc/security/keytabs
its hardcoded in both places

Reply:
I would recommend filing a feature request for the future.  We do have a roadmap item for breaking things out into independent variable files for next quarter.